app/components/InstanceManagerModal/Settings/JavaManagerCard.js

```diff
@@ -0,0 +1,273 @@
+        await promisify(fs.readFile)(
+          path.join(PACKS_PATH, props.instanceName, 'config.json')
+        )
+      );
+      if (config.overrideMemory === undefined && e) {
+        const modifiedConfig = JSON.stringify({
+          ...config,
+          overrideMemory: 4096
+        });
+        await promisify(fs.writeFile)(
+          path.join(PACKS_PATH, props.instanceName, 'config.json'),
+          modifiedConfig
+        );
+        setOverrideJavaMemory(4096);
+        setJavaMemorySwitchState(true);
+        const configChanged = JSON.parse(
+          await promisify(fs.readFile)(
+            path.join(PACKS_PATH, props.instanceName, 'config.json')
+          )
+        );
+        setMemory(configChanged.overrideMemory);
+      } else if (config.overrideMemory !== undefined && !e) {
+        const modifiedConfig = JSON.stringify(_.omit(config, 'overrideMemory'));
+        await promisify(fs.writeFile)(
+          path.join(PACKS_PATH, props.instanceName, 'config.json'),
+          modifiedConfig
+        );
+        setJavaMemorySwitchState(false);
+      }
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  return (
+    <Card style={{ marginTop: 15, height: 'auto' }} title="Java Manager">
+      <div style={{ display: 'inline', verticalAlign: 'middle' }}>
+        <div className={styles.mainTextSlider}>
+          Java Memory (
+          {is64bit ? (
+            '64 bit)'
+          ) : (
+            <span>
+              32 bit)&nbsp;
+              <Tooltip
+                placement="right"
+                title="Your system uses a 32 bit Java, which allows a maximum of 1.5GB to be used.
+                 If you want more, install or select a 64 bit java executable"
+              >
+                <FontAwesomeIcon
+                  className={styles.iconPointer}
+                  icon={faQuestionCircle}
+                />
+              </Tooltip>
+            </span>
+          )}
+          <Switch
+            className={styles.sliderSwitch}
+            onChange={e => toggleJavaMemory(e)}
+            checked={javaMemorySwitchState}
+          />
+        </div>
+        {javaMemorySwitchState ? (
+          <MemorySlider
+            hr={true}
+            overrideJava={true}
+            ram={overrideJavaMemory}
+            is64bit={props.is64bit}
+            updateMemory={updateMemory}
+            javaArguments={overrideArgs}
+            instanceName={props.instanceName}
+            memory={memory}
+          />
+        ) : null}
+      </div>
+
+      <div style={{ display: 'inline', verticalAlign: 'middle' }}>
+        <div className={styles.mainText}>
+          Java Arguments
+          <Switch
+            className={styles.switch}
+            onChange={e => toggleJavaArguments(e)}
+            updateArgs={updateArgs}
+            resetArgs={resetArgs}
+            checked={javaArgsSwitchState}
+          />
+        </div>
+        {javaArgsSwitchState ? (
```

Comment:
use the AND logical operator instead. See https://www.robinwieruch.de/conditional-rendering-react/#logical-and-operator